tests/Faithlife.Analyzers.Tests/NullTestTernariesTests.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Framework;
+
+namespace Faithlife.Analyzers.Tests
+{
+	[TestFixture]
+	public class NullTestTernariesTests : CodeFixVerifier
+	{
+		[Test]
+		public void ValidUsage()
```

Comment:
There should probably be tests with valid ternary operators, to make sure this doesn't incorrectly flag good uses of ternaries.

Reply:
Added some better tests in here.